bindata/manifests/switchdev-config/switchdev-units/NetworkManager.service.yaml

```diff
@@ -3,5 +3,7 @@ dropins:
 - name: 10-switchdev.conf
   contents: |
     [Unit]
-    Wants=switchdev-configuration-before-NM.service
-    After=switchdev-configuration-before-NM.service
+    Wants=switchdev-configuration-before-nm.service
+    Wants=switchdev-configuration-after-nm.service
```

Comment:
Do we need this line?

Reply:
From the testing, yes, this is required to ensure VF netdev driver is binded after reboot

Reply:
@pliurh wdyt?